keyboards/7skb/rev1/rev1.h

```diff
@@ -0,0 +1,46 @@
+#pragma once
+
+#include "7skb.h"
+
+#include "quantum.h"
+#ifdef RGBLIGHT_ENABLE
+//rgb led driver
+#include "ws2812.h"
+#endif
```

Comment:
Not sure this include is needed.

Reply:
Thank you for pointing out.
I fixed it.